### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by lenkon**
### The following are implemented:
- Create an app using create-react-app.
- Install all necessary packages such as react-router, redux, Axios, and other node dependencies.
- Install linters and GitHub workflows.
- Remove all default files.
- Update README.

### CODE REVIEW

File: public/logo512.png
Comment (abel-tefera) [inline public/logo512.png:1]: [Optional] Remove this file as it is not necessary

File: public/logo192.png
Comment (abel-tefera) [inline public/logo192.png:1]: [Optional] You can remove the default react logo from the project

File: src/App.css
Comment (abel-tefera) [inline src/App.css:1]: [Optional] You can clear the contents of this default styling.

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
```
Comment (abel-tefera) [inline src/App.js]: [Optional] Clear the contents of this div

File: src/index.css
Comment (abel-tefera) [inline src/index.css:1]: [Optional] You can clear the contents of the default styling

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```
Comment (abel-tefera) [inline src/App.js]: This function should be refactored to ES6 syntax. So it must be `const App = () => {...}`

**Review by abel-tefera — CHANGES_REQUESTED** [review]
Hi @lenkon, excellent work on setting up the React project. Everything is set up correctly, with just _one_ change that's necessary and others that are optional. 

### Highlights

- React set up properly using CRA
- Linters & github workflows set up
- Readme file done

**Comment by lenkon** [comment]
Hi @abel-tefera ,

Thanks for the feedback. The suggested changes have been implemented.

Regards,
K. Inoulen Shanongba

**Review by abel-tefera — APPROVED** [review]
Hi @lenkon, this PR is approved. There's nothing else to say other than, it's time to merge it.

**Pull request merged**